taichi/backends/cuda/cupti_toolkit.cpp

```diff
@@ -145,7 +145,7 @@ ScopeExit<T> MoveScopeExit(T t) {
 
 #define SCOPE_EXIT(func)                                      \
   const auto NV_ANONYMOUS_VARIABLE_INDIRECT(EXIT, __LINE__) = \
-      MoveScopeExit([=]() { func; })
+      MoveScopeExit([=]() { (void)(func); })
```

Comment:
thanks, I wonder does this fix alone eliminate the warnings?

Reply:
Yes, after the fix all warnings are disappeared.